### ISSUE TITLE
Printing is super tiny...

### ISSUE THREAD

**Issue by ihateu3** [u-235/edytornc#25]
I forgot about this bug.  Whenever I go to print something the print text is so tiny that it is not even legible.  You can see this even in the print preview.  Does anyone know a workaround???
![Screenshot](https://github.com/u-235/edytornc/assets/19616711/e2ebee15-7d57-40c3-8fd2-d256485ffe97)

**Comment by u-235** [comment]
It looks as if the program set the font size in pixels and the printer has a high DPI. But the program sets the font size in points. There is a possibility that the font size could be set in pixels when activating the Qt theme or running the program with the styles specified.

- Does the problem stay when choosing a different printer or when printing to a file?
- No such problem in other programs?

**Comment by ihateu3** [comment]
The problem does still exist when trying another printer.  There are no such problems with other programs.  This should be easily replicable by trying it on your end, you can see right away in the print preview that the text is extremely small before even sending the print job.

**Comment by u-235** [comment]
I found the reason for this behavior. Windows uses several bitmapped fonts such as Courier, MS Sans Serif, MS Serif. These fonts look fine on the screen, but they are not scaled when printed. Perhaps this can be fixed somehow, but for now it is easier to use Courier New.

**Comment by ihateu3** [comment]
Yes your workaround does work!  If you do happen to fix the issue could you also add the option to print the code in black?  Now that I am able to use your work around, I realize that it is printing in the colors I have set in EdytorNC, and the white text on a black background prints as white text on paper, so basically it just doesn't print the white text.

**Comment by u-235** [comment]
#26 - I opened the issue with your suggestions.
